src/rendervulkan.cpp

```diff
@@ -1253,7 +1253,9 @@ static bool init_device()
 	VkPushConstantRange pushConstantRange = {
 		.stageFlags = VK_SHADER_STAGE_COMPUTE_BIT,
 		.offset = 0,
-		.size = uint32_t(sizeof(Composite_t::CompositeData_t) + (g_bIsCompositeDebug ? sizeof(uint32_t) : 0))
+		.size = uint32_t(sizeof(Composite_t::CompositeData_t) + /* composite data */
+				sizeof(uint32_t) + /* border mask */
+				(g_bIsCompositeDebug ? sizeof(uint32_t) : 0)) /* frame id*/
```

Comment:
If you are already changing this, please do it properly. https://github.com/DadSchoorse/gamescope/commit/df59804399234e963b544d8034e005dc40e7c69b

Reply:
Moved this into data to not conflict with your changes and such.